addons/characters/CfgVehicles.hpp

```diff
@@ -0,0 +1,104 @@
+class CfgVehicles {
+    // External class reference
+    class C_Man_casual_1_F_afro;
+    class C_Man_casual_3_F_afro;
+    class C_Man_casual_4_F_afro;
+    class C_Man_casual_5_F_afro;
+    class C_Man_casual_6_F_afro;
+    class C_man_polo_1_F_afro;
+    class C_man_polo_2_F_afro;
+    class C_man_polo_3_F_afro;
+    class C_man_polo_6_F_afro;
+    class C_man_sport_2_F_afro;
+
+    class C_Man_casual_1_F_afro_sick : C_Man_casual_1_F_afro {
+        editorSubcategory = "EdSubcat_Personnel_Sick";
+        scope = 2;
+        scopeCurator = 2;
+        class EventHandlers {
+            init = "(_this select 0) setIdentity selectRandom ['BIS_Ambient01_sick','BIS_Ambient02_sick','BIS_Ambient03_sick','BIS_Arthur_Sick','BIS_Howard_sick','BIS_John_sick','BIS_Lucas_sick','BIS_Renly_sick']; (_this select 0) setDamage 0.45";
```

Comment:
Couldn't all of this be extracted to a define?

Reply:
If only multiple inheritance was a thing...
Do you suggest to move the entire block to a define or just the init line?

Reply:
It doesn't look like they change so maybe all the block? And shouldn't this init line be macro'd?